util/retry/retry.go

```diff
@@ -19,24 +19,17 @@ package retry
 
 import (
 	"fmt"
-	"math/rand"
 	"time"
 
+	"github.com/cenkalti/backoff"
+
 	"github.com/cockroachdb/cockroach/util"
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
-// retryJitter specifies random jitter to add to backoff
-// durations. Specified as a percentage of the backoff.
-const retryJitter = 0.15
-
-// Status is an enum describing the possible statuses of a
-// backoff / retry worker function.
-type Status int32
-
 // MaxAttemptsError indicates max attempts were exceeded.
 type MaxAttemptsError struct {
-	MaxAttempts int
+	MaxAttempts uint
```

Comment:
This change wasn't about the range - it doesn't make any sense for `MaxAttempts` to be negative